espnet/mt/pytorch_backend/mt.py

```diff
@@ -55,22 +56,23 @@
 
 
 class CustomConverter(object):
-    """Custom batch converter for Pytorch
+    """Custom batch converter for Pytorch."""
 
-    :param int idim : index for <pad> in the source language
-    """
-
-    def __init__(self, idim):
+    def __init__(self):
+        """Construct a CustomConverter object."""
         self.ignore_id = -1
-        self.pad = idim
+        self.pad = 0
```

Comment:
I think it would be better to add some comments. For example, `0` is already used in ASR as a blank, and I feel it's a bit risky. Can we set it as `-1`? Or we may at least need some comments.

Reply:
To keep the vocabulary size, I used index:0 for padding. It's OK since blank labels are not used in NMT. The <pad> class is usually treated as a single class in the vocabulary in NLP.